src/common/sidebar-injector.js

```diff
@@ -303,7 +303,7 @@ function SidebarInjector(chromeTabs, dependencies) {
     if (!isSupportedURL(tab.url)) {
       return Promise.resolve();
     }
-    return injectScript(tab.id, '/public/destroy.js');
+    return injectScript(tab.id, '/lib/destroy.js');
```

Comment:
It needs to be somewhere where it can be injected into the page. Nothing in `src/common/` is publicly accessible.